00.xsh

```diff
@@ -1,10 +1,8 @@
-#!/usr/bin/env bash
-
-export START_TIME="$(date +%s)"
-export TIMEOUT=2700
-./setup.sh
+$START_TIME = int("$(date +%s)")
+$TIMEOUT = 2700
+sh setup.sh
```

Comment:
You run setup.sh here, but also in travis_template.yml.

Reply:
Also do you want to run this or `source-bash` it?